dev/io.openliberty.org.eclipse.microprofile/config.3.0.bnd

```diff
@@ -22,6 +22,6 @@ Export-Package: \
   org.eclipse.microprofile.config.spi;version=3.0
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.config:microprofile-config-api;3.0}
+  @${repo;org.eclipse.microprofile.config:microprofile-config-api;3.0.1;EXACT}
```

Comment:
Changing this from an open range (essentially 3.0+) to an exact version makes sense. Using a closed range like [3.0,3.1) would probably also be ok?